packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx

```diff
@@ -171,6 +173,16 @@ function AttachmentPointSelectionPanel({
       return acc;
     }, {});
 
+  const getLeavingGroupValue = (attachmentPoint): LeavingGroupValue => {
+    if (!monomerLeavingGroups) {
+      return 'H';
+    }
+    const leavingGroupValue = monomerLeavingGroups[attachmentPoint];
```

Comment:
I'm not sure about naming here. If anyone could suggest a better term, I would appreciate.